### ISSUE TITLE
The function 'CaptureWrapper' isn't defined.

### ISSUE THREAD

**Issue by theRightHooprod** [joseph-grabinger/flutter_to_pdf#83]
My flutter code can't seems to find this widget.

**Comment by joseph-grabinger** [comment]
I just noticed there is a minor typo in the `README.md`'s example for the `CaptureWrapper`. 

The correct example would be the following: 
```dart
CaptureWrapper(
  key: const Key('someUniqueCustomPaintKey'),
  child: CustomPaint(
    size: const Size(300, 300),
    painter: SomePainter(),
  ),
)
```

This should fix the issue, I'd guess :)

**Comment by joseph-grabinger** [comment]
Closed by #84. Feel free to ask further questions and/or reopen the issue if needed.
